sandbox/lua/encoders/schema_influx.lua

```diff
@@ -139,7 +141,7 @@ function process_message()
 
     local place = 2
     for _, field in ipairs(used_base_fields) do
-        columns[place] = field
+        columns[place] = string.gsub(prepend_fields, "%%{([%w%p]-)}", sub_func)..field
```

Comment:
For performance reasons, this string.gsub action should happen only once, before the fields loop, since it will give the same result every time. Also, it should only happen *if* the user specifies a `prepend_fields` value, we shouldn't waste cycles on a gsub call if nothing was specified.